tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "module": "esnext",
     "lib": ["DOM", "ES2015", "ES2017"],
     "strict": true,
+    "allowJs": true,
```

Comment:
I think it would be better to stick to TS components as this ensures vue-tsc can properly check them. And if it works in TS, it works in JS 🙂 So it would be better to remove this, and convert the script setup to TS I think.